galley/pkg/config/schema/ast/ast_test.go

```diff
@@ -34,23 +34,21 @@ func TestParse(t *testing.T) {
 		{
 			input: `
 collections:
-  - name:         "istio/meshconfig"
-    proto:        "istio.mesh.v1alpha1.MeshConfig"
-    protoPackage: "istio.io/api/mesh/v1alpha1"
+  - name:  "istio/meshconfig"
+    kind:  "VirtualService"
```

Comment:
I am a bit confused why istio/meshconfig has VirtualService?

Reply:
Good catch ... fixed. For parsing the AST test it doesn't really matter much. Fixed